pipeline.yml

```diff
@@ -35,6 +35,14 @@ resources:
         - Dockerfile
       uri: https://github.com/alphagov/cyber-security-concourse-base-image.git
 
+  - name: concourse-base-image-git-amazonlinux2
+    type: git
+    source:
+      branch: amazonlinux2
```

Comment:
Does this need to be changed to master before merging?

Reply:
Yes, this will be changed before being merged.